### PR TITLE
Deploy NuGet package from AppVeyor

### LINKED ISSUE #131: Publish package from CI (AppVeyor)
Publish the NuGet package from AppVeyor, so it's 100% clear where the package is build and which code is used.

### PR DESCRIPTION
**Pull request by 304NotModified**
- change version to 3.0-alpha.1
- needs an encrypted API key

fixes #131

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -39,6 +39,12 @@ for:
     artifacts:
       - path: 'artifacts\*.nupkg'
       - path: 'artifacts\*.snupkg'
+    deploy:
+    - provider: NuGet
+      api_key:
+        secure: <encrypted-nuget-api-key>
```
Comment (304NotModified) [inline appveyor.yml:45]: @axunonb could you please add the **encrypted** api key here?

Steps.
1. Create an API key: https://www.nuget.org/account/apikeys
2. Encrypt the API key here: https://ci.appveyor.com/account/YAXLib/tools/encrypt
3. Edit this file in this PR and replace `<encrypted-nuget-api-key>` with the encrypted API key

Comment (axunonb) [reply appveyor.yml]: @304NotModified done

Comment (304NotModified) [reply appveyor.yml]: Thanks! Looks good!

**Review by axunonb — COMMENTED** [review]
LG

**Comment by 304NotModified** [comment]
As I changes the version to alpha, I think it's save to deploy.

**Review by axunonb — APPROVED** [review]

**Comment by axunonb** [comment]
Excellent, Julian!

**Comment by 304NotModified** [comment]
:)

First alpha live: https://www.nuget.org/packages/YAXLib/3.0.0-alpha.1

I will check the package in depth tomorrow.

**Comment by 304NotModified** [comment]
Green checks!  :) All look good to me :)


image

**Comment by axunonb** [comment]
Very good, had a sneak view yesterday already 😉

**Pull request merged**